src/lib/Components/Inbox/Conversations/Messages.tsx

```diff
@@ -129,13 +136,14 @@ export class Messages extends React.Component<Props, State> {
           width: 708,
           alignSelf: "center",
         }
-      : {}
+      : ({} as any)
```

Comment:
I wonder if it would work without needing to cast to `any` if this were `null` (or `undefined`)?

Reply:
It throws this TS error for `null`, `undefined`, or `{}`:

```
Type '{ inverted: boolean; data: { __id: string; impulse_id: string; is_from_user: boolean; body: strin...' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<FlatListProperties<any>, ComponentState>...'.
  Type '{ inverted: boolean; data: { __id: string; impulse_id: string; is_from_user: boolean; body: strin...' is not assignable to type 'Readonly<FlatListProperties<any>>'.
    Types of property 'style' are incompatible.
      Type '{ width: number; alignSelf: string; }' is not assignable to type 'StyleProp<ScrollViewStyle>'.
        Type '{ width: number; alignSelf: string; }' is not assignable to type 'RecursiveArray<false | ScrollViewStyle | RegisteredStyle<ScrollViewStyle>>'.
          Property 'length' is missing in type '{ width: number; alignSelf: string; }'.
```

Reply:
Interesting! One would think that you could omit `style` for a component. Might be a bug in the RN DT typings? 🤔

Reply:
This is actually  a bug in my VSCode setup - I can run `type-check` with no problems 🤔. Will remove the `any`.